js/components/adPanel.js

```diff
@@ -72,7 +72,16 @@ var AdPanel = React.createClass({
       adPlaybackInfo = adPlaybackInfo + ": (" + currentAdIndex + "/" + totalNumberOfAds + ")";
     }
 
-    var remainingTime = Utils.formatSeconds(Math.max(0, parseInt(this.props.adVideoDuration - this.props.currentPlayhead)));
+    var isLive = this.props.currentAdsInfo.currentAdItem.isLive;
+
+    var remainingTime;
+    if (isLive) {
+      remainingTime = parseInt((this.props.adStartTime + this.props.adVideoDuration * 1000 - new Date().getTime())/1000);
```

Comment:
this.props is guaranteed to exist thanks to React. You can also use defaultProps to guarantee that there's something in that field, rather than adding if statements and null checks everywhere if data consistency is a concern.